src/styles/blocks/card.css

```diff
@@ -0,0 +1,91 @@
+.card {
+  width: 200px;
+  padding: 0 16px;
+  box-sizing: border-box;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  object-fit: contain;
+  height: 134px;
+  width: 160px;
+  margin-top: 32px;
+
+}
+
+.card__name {
+  font-weight: 500;
+  line-height: 18px;
+  align-self: center;
+  margin-top: 40px;
+  margin-bottom: 0;
+  color: #060b35;
+  font-size: 12px;
+  width: 166px;
```

Comment:
You shouldn't specify width for all elements
All block elements has 100% width by default